packages/truffle-contract/lib/utils.js

```diff
@@ -117,49 +141,77 @@ var Utils = {
   },
 
   // Extracts optional tx params from a list of fn arguments
-  getTxParams: function(args){
+  getTxParams: function(methodABI, args) {
```

Comment:
I now pass the `methodABI` through so that it can be used in `utils.getTxParams`